README.md

```diff
@@ -30,6 +30,17 @@ __Capabilities & Limitations__
 
 Nimgen supports compiling in C/C++ sources and static libraries as well as loading in dynamic libraries.
 
+Environment variables are supported via Nim's string interpolation `%` symbol imported from the `strutils` module. Simply use double quotes to enclose any value and put `$` or `${}` around the environment variable name. In addition, the `output` var from the n.global section is available as ${output}. For example:
```

Comment:
Can we move this section to the `Confile File` section?

Reply:
Yep!